src/pod.rs

```diff
@@ -97,26 +100,31 @@ unsafe impl<T: PodTransmutable> PodTransmutable for [T; 32] {}
 ///
 /// An error is raised in one of the following situations:
 ///
-/// - The data does not have a memory alignment compatible with `T`. You will
-///   have to make a copy anyway, or modify how the data was originally made.
 /// - The data does not have enough bytes for a single value `T`.
 ///
+/// # Safety
+/// 
+/// It is undefined behavior If the data does not have a memory alignment
```

Comment:
(a) capitalisation
(b) "it"'s ambiguous, "this (funxion) (invokes?)" works better maybe?

Occurs on:
  * L107
  * L145
  * L264
  * L309